app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -116,6 +117,51 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       }
     },
 
+    populateLinkPathParts: function (clickData) {
+      var href = clickData.url
+      var path = ''
+      if (this.hrefIsRelative(href)) {
+        path = href
+      } else if (this.isMailToLink(href)) {
```

Comment:
Might be worth storing the of `isMailToLink` if possible to prevent having to call it more than once. Not sure it's possible though.

Reply:
See https://github.com/alphagov/govuk_publishing_components/pull/3004#discussion_r990089935